torch/distributed/fsdp/wrap.py

```diff
@@ -308,42 +308,34 @@ class ParamExecOrderState(Enum):
     INITIALIZED = auto()
 
 
-@dataclass
-class ParamExecOrderPolicy:
-    """
-    TODO (awgu): coalesce with below and document
-    """
-    handle_init_mode: HandleInitMode
-
-
 @dataclass
 class ParamExecOrderWrapPolicy:
     """
     This is the class used for the wrapping policy that wraps parameters and performs
     the communication scheduling based on the parameter execution order in the forward pass
     (also called non-recursive wrapping policy).
 
-    The policy contains multiple wraps. Each wrap contains original parameters that will be executed together,
-    and the wrap transfers these parameters into one FlattenParameter. In both forward and the backward passes,
-    the sharded parameters in each wrap will be gathered just before these parameters are used in the passes.
-    These parameters will then be reshaded once they have been used.
-
-    TODO (linjianma): For now, the parameters contained in each wrap of ParamExecOrderWrapPolicy
-    are the parameters in each wrap of the init_policy (a recursive wrapping policy).
-    Later we will wrap parameters based on bucket size.
+    The policy contains multiple ``FlatParamHandle``. Each ``FlatParamHandle`` is a bucket that contains original parameters
+    that will be executed together, and the ``FlatParamHandle`` transfers these parameters into one ``FlattenParameter``.
+    In both forward and the backward passes, the sharded parameters in each ``FlatParamHandle`` will be gathered
+    just before these parameters are used in the passes. These parameters will then be reshaded once they have been used.
 
     Args:
-        init_policy (nn.Module):
-            The initial recursive wrapping policy used to guide the wrapping of this policy. In the first
-            forward and backward iteration, init_policy is used. Parameter execution order is also recorded
-            in the first iteration. Starting from second iteration, ParamExecOrderWrapPolicy will be used.
-
-            The default always_wrap_policy might not be the best choice for every model. For example, for
-            transformer based models, setting transformer_auto_wrap_policy as the init_policy will guarantee
+        handle_init_mode (HandleInitMode):
+            The initial mode used in the policy.
+        bucket_size (int):
+            Bucket size threshold in bytes.
+        module_level_group_policy (Callable):
```

Comment:
Is this always used? Or is it unused if `HandleInitMode` is `PARAM_LEVEL`?

Reply:
yes, this is it unused if `HandleInitMode` is `PARAM_LEVEL`